generators/typescript/utils/core-utilities/fetcher/src/fetcher/getRequestBody.ts

```diff
@@ -1,34 +1,14 @@
-import { RUNTIME } from "../runtime";
-
-export async function getRequestBody(body: any, contentType: string): Promise<BodyInit | undefined> {
-    let requestBody: BodyInit;
-    if (RUNTIME.type === "node") {
-        if (body instanceof (await import("formdata-node")).FormData) {
-            // @ts-expect-error
-            requestBody = body;
-        } else if (body instanceof (await import("stream")).Readable) {
-            // @ts-expect-error
-            requestBody = body;
-        } else {
-            requestBody = maybeStringifyBody(body, contentType ?? "");
-        }
-    } else {
-        if (body instanceof (await import("form-data")).default) {
-            // @ts-expect-error
-            requestBody = body;
-        } else {
-            requestBody = maybeStringifyBody(body, contentType ?? "");
-        }
+export declare namespace GetRequestBody {
+    interface Args {
+        body: unknown;
+        type: string;
```

Comment:
I don't think content type is going to work well here, so probably better if its an enum or we don't do it as part of this PR

Reply:
The reason its not going to work well is because our TypeScript SDKs in some cases don't even send [content type ](https://github.com/elevenlabs/elevenlabs-js/blob/58beeb10866574d797c3adae20001077b7793c4f/src/api/resources/speechToSpeech/client/Client.ts#L70-L94) so it feels not ideal to rely on this being passed in